src/scanner.c

```diff
@@ -124,13 +189,17 @@ bool tree_sitter_pod_external_scanner_scan(
       return false;
 
     if(want_end && c == '>') {
-      ADVANCE;
+      int count = chevron_count_top(state);
+      while(count && c == '>') {
+        ADVANCE_C;
+        count--;
+      }
+      chevron_count_pop(state);
       TOKEN(TOKEN_INTSEQ_END);
```

Comment:
This doesn't do a sanity check that we ate all the chevrons we wanted, leading to a wrong parse for the following input:
`C<<> A<>>>`.  That spits out the following for me:
```
(pod [0, 0] - [1, 0]
  (plain_paragraph [0, 0] - [1, 0]
    (content [0, 0] - [1, 0]
      (interior_sequence [0, 0] - [0, 4]
        letter: (sequence_letter [0, 0] - [0, 1]))
      (interior_sequence [0, 5] - [0, 8]
        letter: (sequence_letter [0, 5] - [0, 6])))))
```

This also raises the question: why is this handled here + also further down in the scanning loop?

Reply:
Ahyes; bug fixed by fff3b6a

Reply:
As to why there's two: We need one set of code (around line 190) that notices a valid TOKEN_INTSEQ_END and emits it only if there is one, and we need a second set of counting code (around line 260) to notice that there's a valid sequence of them and so end the TOKEN_CONTENT_PLAIN at that point, while not ending on a too-short one.

Alternatively we could end the TOKEN_CONTENT_PLAIN at any '>' but then we'd have to emit multiple "content" nodes in such cases.